app/configurator/components/filters.tsx

```diff
@@ -1111,16 +1113,20 @@ export const getTimeFilterOptions = (props: GetTimeFilterOptionsProps) => {
     }[] = [];
     const sortedValues: ObservationValue[] = [];
 
-    for (const { value } of dimension.values) {
-      const date = parse(`${value}`);
+    for (const { value, position } of dimension.values) {
+      const date = temporal ? parse(`${value}`) : parse(`${position}`);
```

Comment:
I think the code would be a bit more robust and understanble if we explicitly check for temporalEntityDimension here and add an exhaustive check ?